CMakeLists.txt

```diff
@@ -50,8 +66,8 @@ include(ROCMInstallSymlinks)
 include(ROCMCreatePackage)
 include(CheckCXXCompilerFlag)
 include(ROCMCheckTargetIds)
-rocm_setup_version(VERSION 0.2.0)
```

Comment:
Let upstream rocm-cmake deal with version. I would not delete this.

Reply:
@JehandadKhan is the version number controlled by the team? I deleted it because there were two versions `1.0.0` and `0.2.0`

Reply:
This is the preferred method by `rocm`